test/channelz_test.go

```diff
@@ -1061,6 +1061,10 @@ func (s) TestCZClientSocketMetricsStreamsAndMessagesCount(t *testing.T) {
 func (s) TestCZClientAndServerSocketMetricsStreamsCountFlowControlRSTStream(t *testing.T) {
 	e := tcpClearRREnv
 	te := newTest(t, e)
+	// Before behavior change in PR #8665, large window sizes set
+	// using InitialWindowSize disabled BDP by default. Post the
+	// behavior change, we have to explicitly disable BDP.
```

Comment:
nit: Can we avoid referencing the PR here and focus only on the intended behaviour? Describing historical context in a test comment might be too detailed and could confuse future readers.